server/scheduler/models.py

```diff
@@ -39,7 +39,13 @@ class Labs(models.Model):
 class Assignment(models.Model):
     scheduleNum = models.AutoField(primary_key = True, editable = False, unique = True)
     semYr = models.ForeignKey('SemesterYear', on_delete=models.CASCADE)
-    id = models.ForeignKey('GATA', on_delete=models.CASCADE)
+    id = models.ForeignKey('GATA', on_delete=models.CASCADE) #GA id which is refernced from GATA
```

Comment:
These id variables need to be distinguished. You are currently overwriting the first instance of "id". These should be named GAid and TAid.